test/unit/plugins/validate-semantic/oas3/refs.js

```diff
@@ -206,4 +206,123 @@ describe("validation plugin - semantic - oas3 refs", () => {
       return expectNoErrorsOrWarnings(spec)
     })
   })
-})
+
+  describe("$refs for headers must reference a header by position", () => {
+    it("should return an error when a header incorrectly references a local parameter schema", () => {
```

Comment:
Test names don't match the validation performed. We're validating all parameters not just header parameters. Please change to
```js
  describe("parameter $refs should not point to header components", () => {
    it("should return an error when a parameter incorrectly references a response header component", () => {
```
or something like that.

Reply:
Done!